bucket/libgen-desktop.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "1.4.1",
+    "description": "Desktop for Library Gensis",
+    "homepage": "https://github.com/libgenapps/LibgenDesktop",
+    "license": "Unknown",
+    "notes": [
+        "Require Windows Vista SP2 or later and .NET Framework 4.5 (Included in Windows 8 or later by default) or higher to run.",
+        "See also https://wiki.mhut.org/software:libgen_desktop for details."
+    ],
+    "pre_install": [
+        "Out-File -FilePath \"$dir\\libgen.config\" -Encoding ascii",
+        "Out-File -FilePath \"$dir\\downloads.json\" -Encoding ascii"
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/libgenapps/LibgenDesktop/releases/download/1.4.1/LibgenDesktop.Portable.64-bit.zip",
+            "hash": "3560006aa20d9be09b6b37cf818364e3b7e0862313a15eb7ce3e8840b2af7c47"
+        },
+        "32bit": {
+            "url": "https://github.com/libgenapps/LibgenDesktop/releases/download/1.4.1/LibgenDesktop.Portable.32-bit.zip",
+            "hash": "2142efa5924cc0d47dc5751cd88892ca4e7dc1beb919c11739f562e94fe05c59"
+        }
+    },
+    "extract_to": "",
+    "shortcuts": [
+        [
+            "LibgenDesktop.exe",
+            "LibgenDesktop"
+        ]
+    ],
+    "persist": [
+        "Mirrors",
+        "Downloads",
+        "libgen.config",
+        "downloads.json"
+    ],
+    "checkver": "github",
```

Comment:
```suggestion
```

I don't think this app is going to get any updates (the github repo is abandoned). So we can remove this field.